modules/nested-elements/widgets/nested-tabs.php

```diff
@@ -0,0 +1,441 @@
+<?php
+namespace Elementor\Modules\NestedElements\Widgets;
```

Comment:
On click on a container in a tab - it doesn't select the container for edit.

Reply:
Handled